test/config.yaml

```diff
@@ -1,10 +1,17 @@
 ---
-logVLevel: 4
-nodeName: node1
-nodeIP: '1.2.3.4'
-cluster:
-  url: https://1.2.3.4:6443
-  clusterCIDR: '10.20.30.40/16'
-  serviceCIDR: '40.30.20.10/16'
-  domain: cluster.local
-  serviceNodePortRange: 30000-32767
+node:
+  nodeName: node1
+  nodeIP: '1.2.3.4'
+core:
+  apiServer:
+    bindHost: https://1.2.3.4:6443
+  network:
+    clusterNetwork:
+      - cidr: '10.20.30.40/16'
```

Comment:
Having a `cidr` field here but not in `serviceNetwork` is one obvious example to me of where we sacrifice UX / intuitiveness of the config API for the sake of alignment.

Reply:
I agreed and it is unfortunate that there is a bit of a difference on how `serviceNetwork` and `clusterNetwork` are constructed on config API side. At this point, it is essentially a choice to make. Do we want to strictly follow the config v1 struct or can we make a slight modification here? @benluddy @deads2k

Reply:
We have agreed to align where possible, so let's just keep this as is.

Reply:
Sounds fair.